app/queries/find_latest_matching_subscription.rb

```diff
@@ -2,9 +2,8 @@ class FindLatestMatchingSubscription
   def self.call(original_subscription)
     subscriber_list_id = original_subscription.subscriber_list_id
     subscriber_id = original_subscription.subscriber_id
-    frequency = original_subscription.frequency
     Subscription.
-      where(subscriber_list_id: subscriber_list_id, subscriber_id: subscriber_id, frequency: frequency).
+      where(subscriber_list_id: subscriber_list_id, subscriber_id: subscriber_id).
```

Comment:
Would it be possible to make the chaining dots consistent over the multiple lines (i.e. at the beginning of the following line)?